epistasis_selection.py

```diff
@@ -58,6 +58,18 @@ def combine_k_rounds(num_rounds: int,
     return all_samples
 
 
+def filter_mutation_sets_for_reference(mutation_sets: Iterable[Tuple[Mutation, ...]],
+                                       reference_seq: Sequence[int]) -> List[Tuple[Mutation, ...]]:
+    """Remove any mutation sets from `mutation_sets` that include a mutation corresponding to the reference sequence."""
+    filtered_mutation_sets = []
+    for mutation_set in mutation_sets:
+        if any([reference_seq[pos] == aa for (pos, aa) in mutation_set]):
```

Comment:
style nit: this continue statement is unecessary and makes it hard to read. Just negate the condition line 66 and do the append right under the if statement.